Project-Bootcamp/views/challenger-view.js

```diff
@@ -0,0 +1,65 @@
+var React = require ('react');
+var _ = require ('lodash');
+var $ = require ('jquery');
+var Button = require ('../components/button');
+var Input = require ('../components/input');
+var Table = require ('../components/table');
+var BlizzardAPI = require ('../querys/query');
+
+var ChallengerView = React.createClass({
+
+    getInitialState: function () {
+
+        return {
+            challengers: []
+        }
+    },
+
+    render: function () {
+
+        return (
+            <div>
+                <div>
+                    <h3><u>Top 10 challenger leaderboard</u></h3>
```

Comment:
Careful here, we should avoid using markup to alter visual appearance. You should style this with CSS.